test/server/modlog.js

```diff
@@ -101,24 +101,26 @@ describe('Modlog', () => {
 		});
 	});
 
-	describe("Modlog#rename", async () => {
-		const entry = {note: 'This is in a modlog that will be renamed!', action: 'UNITTEST'};
+	(Config.usesqlite ? describe : describe.skip)("Modlog#rename", () => {
+		it('should rename modlogs', async () => {
```

Comment:
As far as I can tell this (the `it` change) has equivalent semantics, so if you prefer this way then: sure. This is the kind of change you should just make without requesting review from me, though